Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -0,0 +1,191 @@
+---
+title: CephCluster Network Providers
+---
+
+!!! Note
```

Comment:
Maybe just a personal preference, but I'd suggest to move the note to the end of the section (line ~17) after the intro paragraph.